test/preview/components/rankers/test_transformers_similarity.py

```diff
@@ -1,30 +1,32 @@
 import pytest
 
 from haystack.preview import Document, ComponentError
-from haystack.preview.components.rankers.similarity import SimilarityRanker
+from haystack.preview.components.rankers.transformers_similarity import TransformersSimilarityRanker
 
 
 class TestSimilarityRanker:
     @pytest.mark.unit
     def test_to_dict(self):
-        component = SimilarityRanker()
+        component = TransformersSimilarityRanker(model_name_or_path="cross-encoder/ms-marco-MiniLM-L-6-v2")
```

Comment:
Let's remove the model here so that we test the default parameter values instead.